packages/elements/src/autosuggest/__snapshots__/Autosuggest.md

```diff
@@ -0,0 +1,56 @@
+# `autosuggest/Autosuggest`
+
+#### `DOM structure is correct`
+
+```html
+<div part="header">
+  <slot
+    id="headerSlot"
+    name="header"
+  >
+  </slot>
+</div>
+<div part="content">
+  <slot>
+  </slot>
+  <ef-item
+    aria-selected="false"
+    id="moreResults"
+    part="more-results"
+    tabindex="0"
+  >
+    <span part="more-results-text">
+      More results for
+      <mark>
+      </mark>
+    </span>
```

Comment:
I think we should support A111Y for `more-results-text` item, currently, it does not work.

Reply:
I think probably adding `id` and localised `aria-label` would do.

Reply:
I found one more issue on `more-result` item when pressing `shift` + `enter` keys, the highlighted item will be removed. I think it should keep highlighted item and ready to go on the first item of page 2 by arrow key.

Reply:
We cannot, as currently `More result text` can be changed by the app developer via public static attribute and we are not changing any APIs in this PR. Saying so, that can be added as a separate ticket, and the correct change should be announced.

Reply:
Or perhaps I can try to do something

Reply:
No, unfortunately I cannot translate this string. The issue is that `translate` does not sanitise chunks and the code may be vulnerable to XSS attack. I can use smth like `sanitize-html` library, but that it outside the current scope. Furthermore, more-search item is not pronounced anyway, as active-descendant cannot go outside the DOM scope.

Reply:
OK, I will use old XML solution to sanitise strings :)

Reply:
Does it work?

Reply:
> I found one more issue on `more-result` item when pressing `shift` + `enter` keys, the highlighted item will be removed. I think it should keep highlighted item and ready to go on the first item of page 2 by arrow key.

OK, this issue we will improve later.